client-tools/Import/ImportHelper.cpp

```diff
@@ -889,10 +889,12 @@ void ImportHelper::addField(char const* field, size_t fieldLength, size_t row,
         }
 
         int64_t num = StringUtils::int64(field, fieldLength);
+        size_t bufPos = _lineBuffer.length();
+        _lineBuffer.appendInteger(num);
         if (!_mergeAttributesInstructions.empty()) {
-          lookUpTableValue = std::to_string(num);
+          lookUpTableValue = std::string(_lineBuffer.stringBuffer()->_buffer,
+                                         bufPos, _lineBuffer.length() - bufPos);
```

Comment:
This will call constructor (3) from https://en.cppreference.com/w/cpp/string/basic_string/basic_string, which is potentially unsafe if the lineBuffer is not null-terminated. Let's change it as follows:
```suggestion
          lookUpTableValue = std::string(_lineBuffer.stringBuffer()->_buffer + bufPos,
                                         _lineBuffer.length() - bufPos);
```